### PR TITLE
Merge remote-tracking branch 'origin/feature/logging_abstraction' into task/logging_abstraction/wrapper_usage_refactor

### PR DESCRIPTION
**Pull request by BarAgam**
Changed code to use LoggerWrapper instead of using Android's native logger directly

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/38" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/control/receivers/NewLocationBroadcastReceiver.java [hunk 1]
```diff
@@ -15,14 +16,15 @@
  */
 public class NewLocationBroadcastReceiver extends BroadcastReceiver {
 
-    private static final String LOG_TAG = NewLocationBroadcastReceiver.class.getSimpleName();
+    private static final LogWrapper LOGGER = LogWrapperFactory.create(
```
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/control/receivers/NewLocationBroadcastReceiver.java]: change LOGGER to sLog !fix  and in other classes

**Comment by teamagam1** [comment]
update also LauncherActivity

**Comment by BarAgam** [comment]
done

File: app/src/main/java/com/teamagam/gimelgimel/app/control/receivers/NewLocationBroadcastReceiver.java (hunk 1 above)
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/control/receivers/NewLocationBroadcastReceiver.java]: !fixed

**Comment by teamagam1** [comment]
look for unused TAG (also GGAplication, Base Activity, and so on)

**Pull request merged**